components/esp_lvgl_port/include/esp_lvgl_port.h

```diff
@@ -120,6 +120,8 @@ esp_err_t lvgl_port_stop(void);
  */
 esp_err_t lvgl_port_resume(void);
 
+bool lvgl_port_task_notify(void);
```

Comment:
For what is this function? Any description?

Reply:
This is used to synchronize `RGB` screen refresh and `LVGL tasks`.

If it is an `SPI` screen, the `trans_done` event will not be received if `esp_lcd_panel_draw_bitmap` is not executed. 
This is synchronous, `esp_lcd_panel_draw_bitmap` , and receive `flush_io_ready_callback` event. 
We can handle `lv_disp_flush_ready` in the `trans_done` event.

However, the `esp_lcd_panel_draw_bitmap` and `trans_done` of the `RGB` screen are not generated synchronously. 
The screen is always refreshed and there is always a `trans_done`  event。
So we can only wait for the first event after `draw_bitmap` in `lvgl_port_flush_callback`.